Document changed scheduler behaviour in Version 7 (#5037)
Particular/docs.particular.net: nservicebus/upgrades/6to7/index.md

 * [NServiceBus Testing](../testing-6to7.md)
 * [Azure Blob Storage Data Bus](../absdatabus-1to2.md)

## Scheduled tasks

> In Version 6 scheduled tasks would be invoked as a separate `System.Task` using `Task.Run`.

Reviewer: ```suggestion
In Version 6, scheduled tasks were invoked as a separate `System.Task` using `Task.Run`.
```

> This would mean that:

Reviewer: ```suggestion
This means that:
```

> * They wouldn't be able participate in the [transport transaction](/transports/transactions)

Reviewer: ```suggestion
* They are not able participate in the [transport transaction](/transports/transactions)
```

> * They wouldn't be limited in time by the timeout of the transport transaction

Reviewer: ```suggestion
* They are not limited in time by the timeout of the transport transaction
```

In Version 7 tasks are executed as part of the receive pipeline using an `await`.

> This means that task now will participate in the [transport transaction](/transports/transactions) and is therefor exposed to the transaction timeout.

Reviewer: ```suggestion
This means that the task now participates in the [transport transaction](/transports/transactions) and is therefore exposed to the transaction timeout.
```

> To get the Version 6 behavior please make sure to execute the task in a `Task.Run` as shown below:

Reviewer: ```suggestion
To achieve the Version 6 behavior, please make sure to execute the task in a `Task.Run` as shown below:
```
